java/libphonenumber/pom.xml

```diff
@@ -47,6 +47,14 @@
             <goals>
               <goal>shade</goal>
             </goals>
+            <configuration>
+              <relocations>
+                <relocation>
+                  <pattern>com.google.protobuf</pattern>
+                  <shadedPattern>libphonenumber.repackaged.com.google.protobuf</shadedPattern>
```

Comment:
How about this?

    com.google.i18n.phonenumbers.repackaged.com.google.protobuf